File: counting-DNA/counting.cpp
#include <iostream> 

using namespace std; 

int main(){
    string sequence;
    int numA = 0, numC = 0, numG = 0, numT = 0;

    cin >> sequence;

    for(int i = 0; i < sequence.length(); i++){
        if(sequence.at(i) == 'A'){
            numA++;
        } else if(sequence.at(i) == 'C'){
            numC++;
        } else if(sequence.at(i) == 'G'){
            numG++;
        } else if(sequence.at(i) == 'T'){
            numT++;
        }
    }

    cout << numA << " " << numC << " " << numG << " " << numT << endl;
    return 0;
}

File: counting-DNA/counting.py
sequence = input()

numA, numC, numG, numT = 0, 0, 0, 0

for symbol in sequence:
    if(symbol == "A"):
        numA += 1
    elif(symbol == "C"):
        numC += 1
    elif(symbol == "G"):
        numG += 1
    elif(symbol == "T"):
        numT += 1

print("{} {} {} {}".format(numA, numC, numG, numT))